spec/rubocop/cop/rspec/described_class_spec.rb

```diff
@@ -24,6 +24,26 @@
         end
       RUBY
     end
+
+    it 'ignores offenses within non-rspec numblocks' do
+      expect_offense(<<~RUBY)
+        describe MyClass do
+          controller(ApplicationController) do
+            _1
+            bar = MyClass
```

Comment:
Is this a setter? Doesn’t it need to be prefixed with `self.`?

Reply:
I just moved this over from the test preceeding it. I added the prefix to both to make more sense